### ISSUE TITLE
Use of Jekyll

### ISSUE THREAD

**Issue by jbelien** [OSGeo-be/foss4g.be#3]
I know it will be quite a lot of work but have you ever thought about switching from PHP to [Jekyll](https://jekyllrb.com/).

That's what we use for osm.be and the website can be hosted directly on GitHub if necessary.
It's also easier to update/maintain for non-developer people (basically, it's just Markdown files).

If needed, I would be more than happy to help make the switch !

**Comment by mathieu1** [comment]
I certainly see how a jekyll-based frontend (possibly github-hosted) based on markdown files would be nice. However the website is not fully static, nor is it really blog-like, so it seems to me that in order to have a similar website, the switching costs would also include:
- split the dynamic php code into a [pick your favorite js framework] / [some backend language, maybe still php] combo (with all the deployment tweaks that entails)
- Port all the styling to correctly apply to the markdown files (or have markdown basically be filled with actual HTML)

Both of these items seem somewhat heavy, and I'm not sure the benefits are worth it at this point for a website that needs to be updated once a year. Then again someone more familiar with the setup might have an other opinion.

**Comment by johanvdw** [comment]
The only really dynamic part is the schedule, if someone feels like
tackling the other pages, I can adjust that one.

**Comment by jbelien** [comment]
I had a look at 2017 website and I really think everything could be migrated to Jekyll (included the schedule - it can be based on a JSON file) except the contact page.

If this is something you think worth the shot, I can spend a few hours to do the migration !

**Comment by marcu** [comment]
I totally agree with the idea to have a static website which is generated by jekyll / hugo / pelican, ...

We will investigate this question when the 2022 event will be finish

**Comment by jbelien** [comment]
@marcu Thanks for putting back my suggestion on the table.
I'll be more than happy to help.

**Comment by marcu** [comment]
Oups I forget that subject and create a python version of the website :/
